src/cssModuleHelper.js

```diff
@@ -2,8 +2,10 @@ import path from 'path';
 import vm from 'vm';
 
 const isCssModule = (module) => {
-  const extname = path.extname(module.request);
-  return /\/css-loader\//.test(module.request) && extname !== '.js';
+  if (module && typeof module.request === 'string') {
```

Comment:
can we make this an early return?

`if (!module || typeof module.request !== 'string') return false ...`